biojava-structure/src/main/java/org/biojava/nbio/structure/StructureTools.java

```diff
@@ -1612,12 +1610,9 @@ public static Structure removeModels(Structure s) {
 
 			ChemComp cc = g.getChemComp();
 
-			if (ResidueType.lPeptideLinking.equals(cc.getResidueType())
```

Comment:
This check is not included in `g.isPolymeric()`. Do we need to add it too? I really can't remember why this is important or not. Ideas anyone?